app/pages/explore/[code]/index.vue

```diff
@@ -7,13 +7,12 @@ const pois = ref([])
 const tabs = ref([])
 const initialTab = ref("locations")
 
-// needs fixing for systems
 const starmapLink = computed({
     get() {
         if(location.value) {
-            let link = `https://robertsspaceindustries.com/starmap?location=${location.value.rsi_code}`
-            if (location.value.system) {
-                link = link + `&system=${location.value.system}`
```

Comment:
You removed this whole part? The starmap doesn't work for non-system links without the &system=<something> parameter. Test it for non-system explore pages, you will see the links will be broken.